src/Calculator/SettingsPage.xaml.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "SettingsPage.xaml.h"
+#include "CalcViewModel/Common/AppResourceProvider.h"
+#include "CalcViewModel/Common/LocalizationService.h"
+#include "CalcViewModel/Common/LocalizationStringUtil.h"
+#include "CalcViewModel/Common/TraceLogger.h"
+
+using namespace std;
+using namespace CalculatorApp;
+using namespace CalculatorApp::Common;
+using namespace Platform;
+using namespace Windows::ApplicationModel;
+using namespace Windows::Foundation;
+using namespace Windows::System;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Controls::Primitives;
+using namespace Windows::UI::Xaml::Data;
+using namespace winrt::Windows::Storage;
+
+#ifndef BUILD_YEAR
+#define BUILD_YEAR 2021
+#endif
+
+auto resourceLoader = AppResourceProvider::GetInstance();
```

Comment:
member variables are typically declared in the header file and prefixed with m_ in this project. Also these should be initialized in the constructor instead.